### ISSUE TITLE
Hide Workspace Scheme

### ISSUE THREAD

**Issue by rmnblm** [tuist/tuist#4247]
**Describe the bug**
I am using Tuist 3.0.1 and I have a `Project.swift` with the following options property:

``` swift
options: .options(
  automaticSchemesOptions: .disabled,
  disableBundleAccessors: true,
  disableSynthesizedResourceAccessors: true
)
```

Although no Scheme named `MyProject` is generated, a workspace scheme `MyProject-Workspace` is generated.
What I tried next is to create a `Workspace.swift` in the same folder:

``` swift
import ProjectDescription

let workspace = Workspace(
    name: "CustomWorkspace",
    projects: ["./"],
    schemes: [
        Scheme(
            name: "MyProject-App"
        )
    ],
    generationOptions: .options(
        enableAutomaticXcodeSchemes: false,
        autogeneratedWorkspaceSchemes: .disabled
    )
)
```

However, this did not change anything and the scheme `MyProject-Workspace` is still generated.

What am I doing wrong here? 🥸

**Desktop (please complete the following information):**

- OS: macOS
- Version Monterey

**Comment by adellibovi** [comment]
Hi @rmnblm,
I believe you've been hit by a known bug which was fixed recently in: https://github.com/tuist/tuist/pull/4236

**Comment by adellibovi** [comment]
@rmnblm closing this issue. If the issue is still happening, please reopen the issue.

**Comment by MatyasKriz** [comment]
The options to disable the `-Workspace` scheme generation works in `3.10.0`.

```swift
// Workspace.swift

// Explicitly disabling `-Workspace` scheme generation.
let workspace = Workspace(
    name: "NAME",
    projects: ["./"],
    generationOptions: .options(autogeneratedWorkspaceSchemes: .disabled)
)
```

**Comment by abhi21git** [comment]
@adellibovi Ideally if I am not using multiple projects (which I am not), then I would only like to work with `Project.swift` (don't want to have `Workspace.swift`) and have an option in Project's init to disable generation of .xcworkspace or atleast some flag in Tuist's init in `Tuist.swift`.
TLDR: I don't want .xcworkspace to get generated.
cc: @MatyasKriz @rmnblm
